Src/BootCamp.Chapter/Player.cs

```diff
@@ -103,5 +105,89 @@ public void Equip(Gloves head)
 
         }
         #endregion
+
+        public void AddWeapon(Weapon weapon) 
+        {
+            if (_equipment.GetTotalWeight() > (30 + _strenght * 10))
+            {
+                return;   
+            }
+
+            _equipment.SetWeapon(weapon);
+        }
+
+        public void AddHeadPiece(Headpiece headpiece)
+        {
+            if (_equipment.GetTotalWeight() > (30 + _strenght * 10))
+            {
+                return;
+            }
+
+            _equipment.SetHead(headpiece);
+        }
+
+        public void AddChestPiece(Chestpiece chestpiece)
+        {
+            if (_equipment.GetTotalWeight() > (30 + _strenght * 10))
+            {
+                return;
+            }
+
+            _equipment.SetChest(chestpiece);
+        }
+
+        public void AddArmPiece(Armpiece armpiece, bool isRight )
```

Comment:
All those AddX methods should have a better name- EquipX

Reply:
Also you dont need to say whaty you equip, it's clear from the input type. Use method overloading.

Reply:
oke, so if I understand you , I can name all my function AddEquip or just Equip ?

Reply:
Simply Equip

Reply:
oke, I will change it.